templates/htan-synapse-bucket.yaml

```diff
@@ -113,14 +118,17 @@ Resources:
             Resource: !Sub "${S3Bucket.Arn}/*"
             Condition:
               StringNotEqualsIgnoreCase:
-                s3:x-amz-grant-full-control: !Sub "id=${SynapseCanonicalId},id=${S3CanonicalUserId}"
+                s3:x-amz-grant-full-control: !Sub "id=${SynapseCanonicalId}, id=${S3CanonicalUserId}"
```

Comment:
No space between IDs here, so that this string differs from that in the `StringNotEquals` condition and both will be accepted.

Reply:
done